### PR TITLE
add image transparency and fix issues with gifs

### PR DESCRIPTION
**Pull request by dkanada**
I added the PhotoView dependency as mentioned in #2602 so I could fix the WebView issues and changed the background color to match the current theme.

### CODE REVIEW

**Comment by dkanada** [comment]
I'm guessing you don't want to add jitpack as a new maven repo but I couldn't get it working without that change, so let me know if you have any ideas.

**Comment by bmarty** [comment]
Hi @dkanada ,
Thanks for this new PR. It looks great!
No pb for jitpack dependency.
Would you please add the license for PhotoView in the file `open_source_licenses.html`
Thanks
Benoît

File: vector/src/main/java/im/vector/adapters/VectorMediasViewerAdapter.java [hunk 1]
```diff
@@ -477,34 +448,26 @@ public boolean onLongClick(View v) {
                 height = mMaxImageHeight;
             }
 
-            // the thumbnail is not yet downloaded
-            if (!mMediasCache.isMediaCached(mediaUrl, width, height, mimeType)) {
-                // display nothing
-                container.addView(view, 0);
-                return view;
```
[the remark this answers is not in the document]
Comment (dkanada) [reply vector/src/main/java/im/vector/adapters/VectorMediasViewerAdapter.java]: It was intended, I don't think it makes a difference but adding two views to the container can cause a crash. I was getting that crash a couple times while modifying the code, so I changed that section a bit. It should work the same as before, now there is just one path to add a view to the container and return.

**Comment by dkanada** [comment]
Signed-off-by: Dillon Kanada <dkanada@protonmail.com>

File: vector/src/main/java/im/vector/adapters/VectorMediasViewerAdapter.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (bmarty) [reply vector/src/main/java/im/vector/adapters/VectorMediasViewerAdapter.java]: ok

**Comment by dkanada** [comment]
Somewhat related, I noticed that neither unencrypted nor e2e gifs get thumbnails in the event parameters. However, unencrypted rooms seem to supply a thumbnail while e2e rooms require a complete download. Are there multiple methods to set thumbnails for an event?

**Pull request merged**